XamlControlsGallery/SettingsPage.xaml.cs

```diff
@@ -1,117 +1,130 @@
+                spatialSoundBox.IsChecked = true;
+            if (NavigationRootPage.Current.NavigationView.PaneDisplayMode == Microsoft.UI.Xaml.Controls.NavigationViewPaneDisplayMode.Top)
+                navigationToggle.IsOn = true;
+        }
+
+        protected override void OnNavigatedTo(NavigationEventArgs e)
+        {
+            base.OnNavigatedTo(e);
+
+            NavigationRootPage.Current.NavigationView.Header = "Settings";
+        }
+
+        private async void OnFeedbackButtonClick(object sender, RoutedEventArgs e)
+        {
+            await Launcher.LaunchUriAsync(new Uri("feedback-hub:"));
+        }
+
+        private void OnSettingsPageLoaded(object sender, RoutedEventArgs e)
+        {
+            var currentTheme = ThemeHelper.RootTheme.ToString();
+            (ThemePanel.Children.Cast<RadioButton>().FirstOrDefault(c => c?.Tag?.ToString() == currentTheme)).IsChecked = true;
+        }
+
+        private void OnThemeRadioButtonChecked(object sender, RoutedEventArgs e)
+        {
+            var selectedTheme = ((RadioButton)sender)?.Tag?.ToString();
+            ApplicationViewTitleBar titleBar = ApplicationView.GetForCurrentView().TitleBar;
+
+            if (selectedTheme != null)
+            {
+                ThemeHelper.RootTheme = App.GetEnum<ElementTheme>(selectedTheme);
+            }
+        }
+
+        private void OnThemeRadioButtonKeyDown(object sender, KeyRoutedEventArgs e)
+        {
+            if (e.Key == VirtualKey.Up)
+            {
+                NavigationRootPage.Current.PageHeader.Focus(FocusState.Programmatic);
+            }
+        }
+        private void spatialSoundBox_Checked(object sender, RoutedEventArgs e)
+        {
+            if (soundToggle.IsOn == true)
+            {
+                ElementSoundPlayer.SpatialAudioMode = ElementSpatialAudioMode.On;
+            }
+        }
+
+        private void soundToggle_Toggled(object sender, RoutedEventArgs e)
+        {
+            if (soundToggle.IsOn == true)
+            {
+                spatialSoundBox.IsEnabled = true;
+                ElementSoundPlayer.State = ElementSoundPlayerState.On;
+            }
+            else
+            {
+                spatialSoundBox.IsEnabled = false;
+                spatialSoundBox.IsChecked = false;
+
+                ElementSoundPlayer.State = ElementSoundPlayerState.Off;
+                ElementSoundPlayer.SpatialAudioMode = ElementSpatialAudioMode.Off;
+            }
+        }
+
+        private void navigationToggle_Toggled(object sender, RoutedEventArgs e)
+        {
+            NavigationRootPage.Current.NavigationView.PaneDisplayMode = navigationToggle.IsOn ?
+                Microsoft.UI.Xaml.Controls.NavigationViewPaneDisplayMode.Top :
+                Microsoft.UI.Xaml.Controls.NavigationViewPaneDisplayMode.Left;
+
+            CoreApplication.GetCurrentView().TitleBar.ExtendViewIntoTitleBar = !navigationToggle.IsOn;

```

Comment:
Seeing an issue in dark theme where the minimize/close buttons have the wrong theme colors.

Reply:
Filed a follow up issue for this https://github.com/microsoft/Xaml-Controls-Gallery/issues/403